src/pages/index.js

```diff
@@ -20,88 +64,191 @@ const cardSection = new Section(
   ".gallery__cards"
 );
 
-// Render initial cards
-cardSection.renderItems();
+// Fetch API cards and render them inside the same section
+api.getInitialCards()
+.then((cards) => {
+  cards.forEach((cardData) => {
+    const cardElement = createCard(cardData); // Generate card using createCard
+    cardSection.addItem(cardElement); // Properly add to `.gallery__cards`
+    const likeBtn = document.querySelector('.card__like-button');
+    console.log(cardData.isLiked)
+    if(cardData.isLiked){
+      console.log(cardData._id);  
+      likeBtn.classList.toggle('card__like-button--active');
+    }else{
+      likeBtn.classList.remove('card__like-button--active');
+    }
+  });
+});
```

Comment:
You need to `catch` possible errors at the very **end** of any server request.

**Please, correct it everywhere in the code**